pkg/image/admission/imagepolicy/api/validation/validation.go

```diff
@@ -27,6 +27,12 @@ func Validate(config *api.ImagePolicyConfig) field.ErrorList {
 		}
 	}
 
+	for i, rule := range config.ResolutionRules {
+		if len(rule.TargetResource.Resource) == 0 {
```

Comment:
Remember too that admission needs to be upgraded before you upgrade a cluster - so we actually need to be able to set resource names that don't exist yet (so you don't have a security hole when someone adds a new resource).

Most of these admission controller configs should be translated into namespace and cluster scoped policies very soon.  This will be PodImagePolicy and live in the namespace, and we might want a ClusterPodImagePolicy as well.  For those, we'll want to warn, but again, still can't error out.